### PR TITLE
RU: A few start files

### PR DESCRIPTION
**Pull request by Sinitsaa**
UI & name

### CODE REVIEW

**Comment by alama** [comment]
This is NO good

**Review by alama — CHANGES_REQUESTED** [review]
You can NOT have extra files like ui_craft_RU.csv here

**Comment by alama** [comment]
Make sure you have the correct BASE before making a pull  request

**Comment by Sinitsaa** [comment]
Yes, we have. 
I'm rename "ui_craft_RU.csv" to "ui_craft.csv"

File: translated files/UI/common.csv [hunk 1]
```diff
@@ -0,0 +1,1177 @@
...
+Code658_Message#0,"""所属チームがまだチームルームを入手していないため、<br>移動することができません。<br>"""
+Code659_Title#0,"""チームルーム選択"""
+Code659_Code#0,"""チームルームに移動できませんでした。"""
+Code659_Message#0,"""通信エラーが発生したため、チームルームに<br>移動することができません。<br><br>しばらく時間をおいてから、再びお試しください。"""
+Code660_Title#0,"""チームルーム選択"""
+Code660_Code#0,"""チームルームに移動できませんでした。"""
+Code660_Message#0,"""チームの管理権限を持っていないため、<br>チームルームに移動することができません。"""
+Code661_Title#0,"""チームルーム選択"""
+Code661_Code#0,"""チームルームに移動できませんでした。"""
+Code661_Message#0,"""Не удалось перейти в комнату команды. Не соблюдены условия входа.﻿."""
```
Comment (alama) [inline translated files/UI/common.csv]: Issue with uncode: File UI/common.csv, line 207, position 87, byecode 0xFEFF
ZERO WIDTH NO-BREAK SPACE, AKA, BOM

Comment (Sinitsaa) [reply translated files/UI/common.csv]: ^(=oo=)^
Oho

**Comment by alama** [comment]
Files/co_042800.csv
Files/ui_gacha_001_00.csv
Files/ui_gacha_002_07.csv
Files/ui_gacha_007_09.csv
Files/co_042802.csv
Files/co_042803.csv
Files/ui_gacha_000_00.csv
Files/ui_gacha_001_07.csv
Files/ui_gacha_003_07.csv
Files/ui_gacha_004_09.csv
Files/ui_gacha_008_07.csv

PLEASE move the following files OUT of FILES folder, NO, DELETE them

**Comment by alama** [comment]
You forget to update your branch, I did that for you
now

There can now be ONE file per filename, The folder just there mainly for the user

MOVE  the files, not copy them, delete the 4 files OUT of the Files folder

 common.csv
 ui_craft.csv
 actor_name.csv
 quest_common.csv

**Pull request closed without merging**